.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
I am not able to see running tests in this action :D

Reply:
copy-paste :D